tools/distribution/update_upstream_sdks.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+# -----------------------------------------------------------
+# Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2019-2020 Datadog, Inc.
+# -----------------------------------------------------------
+
+import argparse
+import os
+import re
+import sys
+import traceback
+from tempfile import TemporaryDirectory
+from src.release.semver import Version
+from src.dogfood.repository import Repository
+from src.utils import remember_cwd, shell
+
+
+def update_flutter_sdk(ios_sdk_git_tag: str, dry_run: bool):
+    """
+    Updates `dd-sdk-ios` version in `dd-sdk-flutter` by changing `s.dependency` in `ios/datadog_sdk.podspec`:
+    ```
+      s.dependency 'DatadogSDK', '1.8.0'
+      s.dependency 'DatadogSDKCrashReporting', '1.8.0'
+    ```
+    """
+    with TemporaryDirectory() as clone_dir:
+        print(f'ℹ️️ Changing current directory to: {clone_dir}')
+        os.chdir(clone_dir)
+
+        # Clone repo and create branch:
+        flutter_repo_name = 'dd-sdk-flutter'
+        repository = Repository.clone(
+            ssh='git@github.com:DataDog/dd-sdk-flutter.git',
+            repository_name=flutter_repo_name,
+            temp_dir=clone_dir
+        )
+        repository.create_branch(f'update-dd-sdk-ios-to-{ios_sdk_git_tag}')
```

Comment:
Thinking a little more about this, can we change this branch to `update/dd-sdk-ios-to-{ios_sdk_git_tag}`?